cmd/agent/dist/conf.d/snmp.d/auto_conf.yaml

```diff
@@ -119,3 +119,9 @@ instances:
     ## Changing namespace will cause devices being recreated in NDM app.
     #
     namespace: "%%extra_namespace%%"
+
+    ## @param use_device_id_as_hostname - boolean - optional - default: false
+    ## Use an hash of a device ip address and the namespace as `hostname` for metrics and
```

Comment:
```suggestion
    ## Use an hash of a device IP address and the namespace as `hostname` for metrics and
```

Reply:
No idea why I marked this as resolved, sorry for this